schemas/clinvar-study.tables.json

```diff
@@ -0,0 +1,3068 @@
+          }
+        ]
+      },
+      {
+        "name": "variation_archive_type",
+        "columns": [
+          {
+            "name": "persistent_id",
+            "datatype": "string"
+          },
+          {
+            "name": "record_status",
+            "datatype": "string"
+          },
+          {
+            "name": "replaced_by",
+            "datatype": "string"
+          },
+          {
+            "name": "replaced_list",
+            "datatype": "string"
+          },
+          {
+            "name": "comment",
+            "datatype": "string"
+          },
+          {
+            "name": "species",
+            "datatype": "string"
+          },
+          {
+            "name": "interpreted_record",
+            "datatype": "string"
+          },
+          {
+            "name": "included_record",
+            "datatype": "string"
+          },
+          {
+            "name": "variation_id",
+            "datatype": "integer"
+          },
+          {
+            "name": "variation_name",
+            "datatype": "string"
+          },
+          {
+            "name": "variation_type",
+            "datatype": "string"
+          },
+          {
+            "name": "date_created",
+            "datatype": "date"
+          },
+          {
+            "name": "date_last_updated",
+            "datatype": "date"
+          },
+          {
+            "name": "accession",
+            "datatype": "string"
+          },
+          {
+            "name": "version",
+            "datatype": "integer"
+          },
+          {
+            "name": "number_of_submitters",
+            "datatype": "integer"
+          },
+          {
+            "name": "number_of_submissions",
+            "datatype": "integer"
+          },
+          {
+            "name": "record_type",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "type_record_history",
+        "columns": [
+          {
+            "name": "persistent_id",
+            "datatype": "string"
+          },
+          {
+            "name": "comment",
+            "datatype": "string"
+          },
+          {
+            "name": "accession",
+            "datatype": "string"
+          },
+          {
+            "name": "version",
+            "datatype": "integer"
+          },
+          {
+            "name": "date_changed",
+            "datatype": "date"
+          },
+          {
+            "name": "variation_id",
+            "datatype": "integer"
+          }
+        ]
+      },
+      {
+        "name": "type_comment",
+        "columns": [
+          {
+            "name": "persistent_id",
+            "datatype": "string"
+          },
+          {
+            "name": "data_source",
+            "datatype": "string"
+          },
+          {
+            "name": "type",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "type_comment_type",
```

Comment:
Would consider dropping this table, and changing the `type` column in the `comment` table to just be `comment`

Reply:
Partially agree. We should remove the `type_comment_type` table since this is effectively just an enumeration of possible values. But the `type` field needs to stay and it should hold the enumerated value as a string, and we should add the `comment` field to `type_comment`. 

Here's an example of what the XML data looks like, maybe this will help a little:
`<Comment DataSource="ClinGen" Type="public">ClinGen staff contributed the HGVS expression for this variant.</Comment>`

So you can see we need to preserve the `data_source`, `type` and `comment` in `type_comment`, but can get rid of `type_comment_type`